app/helpers/registered_applications_helper.rb

```diff
@@ -0,0 +1,2 @@
+module RegisteredApplicationsHelper
+end
```

Comment:
Reduce clutter build up, consider deleting unused helper (try to avoid `git add`-ing these).